store/tikv/2pc.go

```diff
@@ -119,6 +119,15 @@ func (c *twoPhaseCommitter) initKeysAndMutations(txn *tikvTxn, connID uint64) er
 		lockCnt int
 	)
 	mutations := make(map[string]*mutationEx)
+	if txn.IsPessimistic() {
+		keys = append(keys, c.primaryKey)
```

Comment:
primaryKey is not in the mutations ?

Reply:
We need to put it in the first position of `keys` because there are many places assume the primary key is the first key.